package.json

```
@@ -28,32 +28,32 @@
     "deploy": "gh-pages -d example/build"
   },
   "peerDependencies": {
-    "react": "^16.0.0"
+    "react": "^17.0.2"
```

As suggested in issue #49 it may be best to allow compatibility with both 16 and 17.
```suggestion
    "react": "^16 || ^17"
```